spark/v3.5/spark/src/main/java/org/apache/iceberg/spark/source/SparkView.java

```diff
@@ -0,0 +1,133 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.iceberg.spark.source;
+
+import static org.apache.iceberg.TableProperties.FORMAT_VERSION;
+
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+import org.apache.iceberg.relocated.com.google.common.collect.ImmutableMap;
+import org.apache.iceberg.relocated.com.google.common.collect.ImmutableSet;
+import org.apache.iceberg.spark.SparkSchemaUtil;
+import org.apache.iceberg.view.BaseView;
+import org.apache.iceberg.view.View;
+import org.apache.iceberg.view.ViewOperations;
+import org.apache.spark.sql.types.StructType;
+
+public class SparkView implements org.apache.spark.sql.connector.catalog.View {
+
+  private static final Set<String> RESERVED_PROPERTIES =
+      ImmutableSet.of("provider", "location", FORMAT_VERSION);
+
+  private final View icebergView;
+
+  public SparkView(View icebergView) {
+    this.icebergView = icebergView;
+  }
+
+  public View view() {
+    return icebergView;
+  }
+
+  @Override
+  public String name() {
+    return icebergView.toString();
```

Comment:
Why not `icebergView.name()`?

Reply:
I've kept this in-sync with `SparkTable`: https://github.com/apache/iceberg/blob/afe4aec4db795f4829757d738a5bbcf1b7db8fd2/spark/v3.5/spark/src/main/java/org/apache/iceberg/spark/source/SparkTable.java#L169

Reply:
I was thinking we would probably want to behave them both the same way. So if we update `SparkView` we should maybe also update `SparkTable`?

Reply:
I looked at BaseTable and BaseMetadataTable, their `toString` just delegates to `name`. I don't have an opinion for views beyond it should be consistent in terms of either it's a fully qualified relation name or not (so if the name() is fully qualified in Table it should be the same in view. Or if it's not fully qualified, it's the same in View)

Reply:
I think we added `name` later. We should use `name` instead here.

Reply:
I've updated this to use `name`